terraform-plans/main.tf

```diff
@@ -6,34 +6,34 @@ module "github_settings" {
   branch                 = var.branch
 }
 
-module "github_workflow_files" {
-  source         = "./modules/GitHub/workflows"
+module "github_templates" {
+  source         = "./modules/GitHub/templates"
   owner          = var.owner
   repository     = var.repository
   branch         = var.branch
-  workflow_files = var.workflow_files
+  templates = var.templates
```

Comment:
nitpick, different space with previous line, please align them if possible. Please solve the conversation by yourself after fix it.

Reply:
Thanks, done. :)